exasol/error/_parse.py

```diff
@@ -203,12 +270,20 @@ def errors(self) -> Iterable["Validator.Error"]:
     def warnings(self) -> Iterable["Validator.Warning"]:
         return self._validator.warnings
 
-    def _make_error(self, node: ast.Call) -> ErrorCodeDetails:
-        code: ast.Constant
-        message: ast.Constant
-        mitigations: Union[ast.List, ast.Constant]
-        parameters: ast.Dict
-        code, message, mitigations, parameters = _extract_parameters(node)
+    def _construct_mitigations(self, code: ast.Constant, mitigations: Union[ast.Constant, ast.List]):
+        if isinstance(mitigations, ast.Constant):
+            return [mitigations.value]
+        if isinstance(mitigations, ast.List):
+            mitigation_values = [m.value for m in mitigations.elts if isinstance(m, ast.Constant)]
+            if len(mitigation_values) != len(mitigations.elts):
+                raise Exception(f"Internal error: validation not correct for '{code}'. Not all elements in list are constants.")
```

Comment:
I think, for that, we need one of these StaticErrors because they are the Error Code used in this project. Basically, the project can't analyze it self.

Reply:
Not needed anymore. The `_make_error` cannot run into this error after the latest refactoring.